src/widgetastic_patternfly/__init__.py

```diff
@@ -1888,6 +1888,62 @@ def items(self):
         return items
 
 
+class Modal(Widget):
+    """ Patternfly modal widget
+
+        https://www.patternfly.org/pattern-library/widgets/#modal
+    """
+    ROOT = ParametrizedLocator('.//div[normalize-space(@id)={@id|quote} and '
+                               'contains(@class, "modal") and contains(@class, "fade") '
+                               'and @role="dialog"]')
+    CLOSE_LOC = './/div[@class="modal-header"]/button[@class="close"]'
+    # widgets for the title, cancel, submit, form widget
+    title_widget = Text(locator='.//h4[@class="modal-title"]')
+    cancel_button = Button(classes="btn-default")
+    submit_button = Button(classes="btn-primary")
+    form_widget = None
+
+    def __init__(self, parent, id=None, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+        self.id = id
+        if not self.id:
+            # if no id, we make assumptions about the modal's locator
+            self.ROOT = ('.//div[contains(@class, "modal") '
+                         'and contains(@class, "fade") and @role="dialog"]')
+
+    @property
+    def title(self):
+        return self.title_widget.read()
+
+    @property
+    def is_open(self):
```

Comment:
do you think we need two same properties ?

Reply:
Nope, I was copying structure from `AboutModal` but I can get rid of this.